src/screens/auth/signup/steps/IdStep.tsx

```
@@ -1,82 +1,35 @@
-          <RecommandIdContainer>
-            <Typography children='이런 아이디는 어떠세요?' font='semiBold14' color='recommandIdText' />
-            <Typography children={recommandId} font='semiBold16' color='defaultWhite' decoration={true} />
-            <ClickText>
-              <Typography children='클릭하여 바로 입력' font='regular12' color='defaultGray' />
-            </ClickText>
-          </RecommandIdContainer>
-        </Pressable>
-      }
-    </Continaer>
+    <ContentLayout>
+      <Typography children='이메일 입력' font='medium28' color='defaultBlack' />
```

Biome (2.4.6)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)